lib/sprockets/commoner/processor.rb

```diff
@@ -47,6 +47,15 @@ def self.call(input)
         instance(input[:environment]).call(input)
       end
 
+      def self.configure(env, *args, **kwargs)
+        env.postprocessors['application/javascript'].each do |processor|
```

Comment:
it will be, as `postprocessors` has a default value https://github.com/rails/sprockets/blob/master/lib/sprockets.rb#L21